tutorials/meshing_of_fractures.ipynb

```diff
@@ -56,10 +56,11 @@
    ]
```

Comment:
<p>I don't see this as the most interesting feature. I think we could remove the export from the tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>